src/main/java/uk/gov/register/presentation/Cardinality.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.register.presentation;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public enum Cardinality {
+    ONE("1"),MANY("n");
+
+    private final String id;
+
+    Cardinality(String id) {
+        this.id = id;
+    }
+
+    private static final Map<String,Cardinality> cardinalityMap = ImmutableMap.of(
+            ONE.getId(),  ONE,
+            MANY.getId(), MANY
+    );
+
+    /**
+     * this method shouldn't be needed -- the @JsonValue annotation has a special case for enums that
+     * means it can be used for both serialization and deserialization -- but there's a bug.
+     * See <a href="https://github.com/dropwizard/dropwizard/issues/699">dropwizard issue #699</a>
+     */
+    @JsonCreator
+    public static Cardinality fromId(String id) {
+        return cardinalityMap.get(id);
```

Comment:
I'm ambivalent about this. Happy to change it if you prefer it but I'm not bothered.